### ISSUE TITLE
Please add a 'release' here on github

### ISSUE THREAD

**Issue by yereverluvinunclebert** [JitHubApp/JitHubV2#61]
It is all very well having the Microsoft Store handle all releases but that is no good for those that do not use the Microsoft store for what ever reason. I do not.

You should be able to add your installer to be a release on github. That would provide us open-sourcers with a much more preferable download location. 

For your information, the second I saw your release was only available from the MS store I decided to look at other alternatives. In the end I find I would rather spend time opening an issue here than visit the Microsoft Store even if it means I will never test your tool.

My team and I use github desktop and similar tools on several operating systems ranging from ReactOS/XP to Win 11 (mostly win 7/10 &11)  and various flavours of Linux, I am looking for an alternative that caters for lots of o/s variations/flavours.

**Comment by roke-julian-lockhart** [comment]
> https://github.com/JitHubApp/JitHubV2/issues/61#issue-2474158382

@yereverluvinunclebert, I agree. We can of course install the application from https://store.rg-adguard.net/, but that's not a particularly trustworthy source.

Additionally, for this to be published on `winget`'s repository, there needs to be a vanity link for the latest release, which GitHub Releases provides.

**Comment by nerocui** [comment]
That's insightful. Thanks for sharing that. I have limited time spent on JitHub, and there's a massive re-architecturing going on. But I'll will keep your request in mind and produce a open source friendly release on GitHub when the time comes.
